ajax/libs/simple-slider/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "simple-slider",
+  "version": "1.0.0",
+  "description": "The 1kb JavaScript Carousel",
+  "npmName": "simple-slider",
```

Comment:
hi @pvnr0082t, many thanks for taking the time to review this!

honestly the missing versions from npm are very very old and not the most stable, I'm really interested in making available versions greater than `1.0.0` which was the first stable release